src/domain/event/eventForm/VirtualEventCheckboxField.tsx

```diff
@@ -11,16 +11,17 @@ const VirtualEventCheckboxField = ({
   form,
   ...props
 }: FieldProps & CheckboxProps) => {
-  const { value: isVirtual } = props.field;
   const { setFieldValue } = form;
 
-  React.useEffect(() => {
-    if (isVirtual) {
+  const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFieldValue('isVirtual', e.target.checked);
+    if (e.target.checked) {
       setFieldValue('location', '');
+      setFieldValue('isOrderable', false);
```

Comment:
Would bookable be a better term? I first thought this PR was something about reordering the checkboxes :)

Reply:
I agree